### PR TITLE
Demix NavigatorConcurrentHardware

### PR DESCRIPTION
**Pull request by foolip**
Part of https://github.com/mdn/browser-compat-data/issues/8929

### CODE REVIEW

**Comment by foolip** [comment]
This is simply splitting the entry without changes. I checked the data against mdn-bcd-collector results and didn't find any differences for Worker. For WorkerNavigator there was a proposed change, but it doesn't look legit, I filed https://github.com/foolip/mdn-bcd-collector/issues/960.

**Comment by foolip** [comment]
@Elchi3 changes like this will require a lot of fixes on MDN. I'd like to not volunteer for that, but is it useful for me to send PRs like this one?

File: api/Navigator.json [hunk 1]
```diff
@@ -1351,6 +1351,56 @@
           }
         }
       },
+      "hardwareConcurrency": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerNavigator/hardwareConcurrency",
```
Comment (Elchi3) [inline api/Navigator.json:1356]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/hardwareConcurrency",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

**Comment by Elchi3** [comment]
> @Elchi3 changes like this will require a lot of fixes on MDN. I'd like to not volunteer for that, but is it useful for me to send PRs like this one?

I'm trying to burn through all mixins on BCD and on MDN. Any help is useful :)

File: api/Navigator.json (hunk 1 above)
Comment (foolip) [reply api/Navigator.json:1356]: Oops, thanks!

**Comment by foolip** [comment]
Before this is merged I want to note that `WorkerNavigator` and `WorkerLocation` are odd cases and it's not obvious how MDN/BCD should represent them. These interfaces predate the use of `[Exposed=(Window,Worker)]` and are subset copies of `Navigator` and `Location`.

The thing is that even though these interfaces are real, web developers just use `navigator.language` or `location.href` in both window and worker scripts. I'm skeptical that it makes sense to document them separately, and am unsure if spitting them on BCD is the right thing.

**Comment by foolip** [comment]
Or rather I should say I could see us changing this later to use the same mechanism for representing global scope exposure as for interfaces and global methods, assuming a single method for that emerges after `WindowOrWorkerGlobalScope` demixing.

**Comment by Elchi3** [comment]
hm, I see. Can we file a separate issue that discusses options for the whole Window/Worker problem with mixins? We can wait to merge this PR until we have had that discussion if you like.

**Comment by foolip** [comment]
I've filed https://github.com/mdn/browser-compat-data/issues/9125 for this specific issue, and https://github.com/mdn/browser-compat-data/issues/9127 for `WindowOrWorkerGlobalScope`.

**Comment by foolip** [comment]
I've rebased this now but I don't like it. It's a similar issue to demixing `WindowOrWorkerGlobalScope`, where web developers can use the same code in windows and workers, and splitting it into two entries seems unhelpful. Finding a way to represent this as one entry meaning `navigator.hardwareConcurrency` and using notes or `partial_implementation` if it doesn't work in workers seems better.

**Comment by Elchi3** [comment]
Thanks Philip! I propose we sort out all other mixins first and then look at everything that touches the window and worker problematic.

**Comment by Elchi3** [comment]
This was done in https://github.com/mdn/browser-compat-data/pull/11413

**Pull request closed without merging**